docs/source/usage/index.rst

```diff
@@ -225,6 +225,44 @@ In order to use an external provisioning service, simply attach the node to the
 | Attach Network to Node        | ``openstack esi node network attach (--network <network> | --port <port>) <node>`` |
 +-------------------------------+------------------------------------------------------------------------------------+
 
+Serial Console Access
+---------------------
+
+Ironic nodes can be accessed through configuring a web or serial interface.
+
+In order to access a node using a serial console, the admin or owner must configure the node's console interface.
+Instructions for this can be found _here. https://docs.openstack.org/ironic/latest/admin/console.html
```

Comment:
The actual link goes at the bottom of the page; look at how the links for, say, Metalsmith are done in this document,